### PR TITLE
BAH-3030 | Added global prop for registration

### PR DESCRIPTION
**Pull request by atish160384**
Added global properties like sms template and clinic timings to be used when patient is registered

### CODE REVIEW

File: masterdata/configuration/globalproperties/gp_registration.xml [hunk 1]
```diff
@@ -47,6 +47,14 @@ where v.date_stopped is null AND v.voided = 0;</value>
       <property>clinic.helpDeskNumber</property>
       <value>+919999999999</value>
     </globalProperty>
+    <globalProperty>
+    <property>clinic.clinicTimings</property>
+      <value>Mon-Fri (09:00-22:00), Sat-Sun CLOSED</value>
+    </globalProperty>
+    <globalProperty>
+      <property>bahmni.registrationSMSTemplate</property>
```
Comment (gsluthra) [inline masterdata/configuration/globalproperties/gp_registration.xml]: There isn't a section for "sms" configuration in global properties?

Comment (atish160384) [reply masterdata/configuration/globalproperties/gp_registration.xml]: Yah there is but most of the templates were there in bahmni section sms section contain mostly the global properties to enable sms for (appointment booking. appointment reminder ,patient registration) we can refractor it to sms if required

Comment (gsluthra) [reply masterdata/configuration/globalproperties/gp_registration.xml]: I see. Would be good to group all SMS related properties in one SMS section. Else its confusing for implementors that which section to they need to see for SMS, and can forget changing something. If all properties are grouped togther, it makes it easy to see.

Comment (atish160384) [reply masterdata/configuration/globalproperties/gp_registration.xml]: Sure Gurpreet will do the changes

**Comment by atish160384** [comment]
New changes will be pushed into 2962 branch

**Pull request closed without merging**